### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by vaihtovirta**
Russian translation for 1.3.0
Fix for issue #153

### CODE REVIEW

File: version/1/3/0/ru/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Кодекс поведения автора
+
+Как авторы и разработчики этого проекта, а также в интересах содействия открытому
+и дружелюбному сообществу, мы обязуемся уважать всех людей,
+которые вносят вклад, сообщая о проблемах, оставляя запросы новой
+функциональности, обновляя документацию, посылая пулл реквесты и патчи,
+а также с помощью другой деятельности.
```
Comment (novel) [inline version/1/3/0/ru/code_of_conduct.md]: "с помощью другой деятельности" sounds weird. Maybe just "или другим образом"?

**Comment by maxim** [comment]
Looks solid to me :+1:

File: version/1/3/0/ru/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Мы стремимся сделать участие в этом проекте свободным от притеснения
+опытом для всех, независимо от уровня опыта, пола, половой идентичности и выражения,
```
Comment (novel) [inline version/1/3/0/ru/code_of_conduct.md]: "притесения опытом для всех" doesn't sound Russian. I'd just drop that, i.e. "свободным от пртеснения, независимо .."

File: version/1/3/0/ru/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+или отклонять комментарии, коммиты, код, изменения в вики,
+вопросы и другой вклад, который не совпадает с настоящим Кодексом Поведения,
+или забанить временно или бессрочно любого участника за любое другое поведение,
+которые они считают нецелесообразным, угрожающим, оскорбительным или вредным.
+
+Приняв этот кодекс поведения, разработчики проекта берут на себя обязательство
+справедливо и последовательно применять эти принципы к каждому
+аспекту управления этим проектом.
+Разработчики, которые не следуют или не обеспечивают
+соблюдение Кодекса Поведения могут быть удалены из команды проекта.
```
Comment (novel) [inline version/1/3/0/ru/code_of_conduct.md]: I think there should be a comma after "Поведения".

Comment (maxim) [inline version/1/3/0/ru/code_of_conduct.md]: yes

File: version/1/3/0/ru/code_of_conduct.md (hunk 1 above)
Comment (pavloo) [inline version/1/3/0/ru/code_of_conduct.md]: agreed

File: version/1/3/0/ru/code_of_conduct.md (hunk 2 above)
Comment (vaihtovirta) [inline version/1/3/0/ru/code_of_conduct.md]: Good point, thank you.

**Comment by vaihtovirta** [comment]
Hey guys, thank you for your comments! I've updated this PR with fixes which you've suggested.
Please let me know if there's anything that can be improved.

**Comment by CoralineAda** [comment]
Thanks so much Emil, and everyone who helped! ♥️ you all.

**Pull request merged**